v2/cmd/voucher_server/TUTORIAL.md

```diff
@@ -14,12 +14,7 @@ Make sure that you update the `ejson` or `sops` specific blocks to point to the
 
 ## Create secrets
 
-<<<<<<< HEAD
-(Deprecated, clair is no longer supported)
```

Comment:
Would we want to mention somewhere that `clair` is no longer supported?

Reply:
We were OK with removing the mention in https://github.com/grafeas/voucher/pull/66 ; so I don't think it's a big issue.

I think it's easy enough to see this being removed - https://github.com/search?q=repo%3Agrafeas%2Fvoucher%20clair&type=code , I don't feel bad not leaving a not that a feature that nobody used is no longer supported 😆 .